### ISSUE TITLE
Accessibility: Support for the number of results found for the screen reader.

### ISSUE THREAD

**Issue by MarkoTP** [microsoft-search/pnp-modern-search#4022]
**Version used**
4.13.1

**Describe the bug**
A few PnP Modern Search issues were identified during the accessibility audit.
Accessibility: Using the PnP Modern Search box and Search Results -webparts, after searching, the screen reader (e.g., NVDA) should inform the user of the search results, for example, “X results found” (alternatively “No results or 0 results found” if there are no results), using the current language resource text. The same issue occurs when using a mobile device. PnP - Search Results is configured to the People layout.
WCAG 2.1, 4.1.3.

**To Reproduce**
Detailed steps to reproduce the behavior:
Configure search page you like with search box and results webparts. Change People layout to result webpart. "Show results count" switch is on.
Execute a search using a person’s name that cannot be found and a person’s name that has multiple entries.

**Expected behavior**
The screen reader (e.g., NVDA) should inform the user of the search results, for example, “X results found” (alternatively “No results” if there are no results), using the current language resource text.

**Desktop (please complete the following information):**
Browser: Chrome, Version 129.0.6668.90 (Official Build) (64-bit)
